source/set_up_3dsecure/index.html.md.erb

```diff
@@ -1,56 +1,70 @@
 ---
-title: Set up 3D Secure
+title: Set up Strong Customer Authentication (3DS1 and 3DS2)
 last_reviewed_on: 2020-06-10
 review_in: 3 months
 weight: 170
 ---
 
-# Set up 3D Secure (3DS)
+# Set up Strong Customer Authentication (3DS1 and 3DS2)
 
-How you set up 3DS payment authentication depends on:
+Strong Customer Authentication (SCA) makes online payments more secure for users and reduces fraud. SCA is part of the Payment Service Directive 2 (PSD2).
 
-- which Payment Service Provider (PSP) you're using
-- if you want to set up 3D Secure 2 (3DS2), which adds an extra layer of identity confirmation before your user pays
+You must implement SCA by:
 
-## Stripe
+- 31 December 2020 if you take payments from the European Economic Area (EEA), defined as the European Union, Norway, Iceland and Liechtenstein
+- 14 September 2021 if you only take payments from within the UK
 
-You do not need to do anything, because we set up 3DS2 on your service by default.
+To implement SCA, you must set up 3D Secure 2 (3DS2) or 3D Secure 1 (3DS1) payment authentication.
 
-## Worldpay
+How you set up 3DS2 or 3DS1 payment authentication depends on which Payment Service Provider (PSP) you’re using.
 
-### Set up 3DS
+## Set up SCA with Stripe
 
-1. Ask your Worldpay account manager to set up 3DS on your merchant code for all payments.
-2. Once Worldpay have set up your merchant code, select the account you want to set up on the [__My services__](https://selfservice.payments.service.gov.uk/my-services) page in the GOV.UK Pay admin tool.
-3. In the settings, select the __View__ link under __3D Secure__.
-4. Select __On__.
+You do not need to do anything because we set up 3DS2 on your service by default.
+
+Stripe automatically applies exemptions.
+
+## Set up SCA with Worldpay
+
+Worldpay sends all transactions using 3DS2 or 3DS1. 3DS Flex is the Worldpay product that enables 3DS2 payment authentication. You must pay a fee to Worldpay to use 3DS Flex.
+
+Worldpay sends you your 3DS Flex credentials when you [go live](/switching_to_live/set_up_a_live_worldpay_account/#connect-your-live-account-to-worldpay).
+
+Worldpay does not apply any exemptions. Contact GOV.UK Pay to apply for exemptions for your transactions.
```

Comment:
I think we discussed this but maybe we decided against changing anything. Pay doesn't support exemptions on Worldpay at the moment; we'd need to build to something to support this. Could we say something like 'contact Pay to ask about exemptions' or 'to discuss exemptions'? just don't want users to think we can do anything immediately...

Reply:
@miriamalice if we want to make sure users don't think they can do anything with exemptions, we could remove that sentence entirely?